Flickr/Sources/FlickrAppVCs/FileManagerCoordinator.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  FileManagerCoordinator.swift
+//  Flickr
+//
+//  Created by Ananth Desai on 01/04/22.
+//
+
+import Foundation
+
+class FileManagerCoordinator {
```

Comment:
You can use the name `PersistenceManager` here. As this is not associated with any view directly and is not responsible for the presentation of views it is not a coordinator.